### PR TITLE
YETUS-1008. Add .asf.yaml to allow GitHub to JIRA integration

### PR DESCRIPTION
**Pull request by aajisaka**
JIRA: https://issues.apache.org/jira/browse/YETUS-1008

### CODE REVIEW

File: .asf.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+---
+github:
+  enabled_merge_buttons:
+    squash: true
+    merge: false
+    rebase: false
+notifications:
+  commits: commits@yetus.apache.org
+  issues: notifications@yetus.apache.org
+  pullrequests: notifications@yetus.apache.org
```
Comment (aw-was-here) [inline .asf.yaml:25]: I think most of this goes to gitbox@ currently.

Comment (aajisaka) [reply .asf.yaml:25]: Oh, I didn't know that ML.  I'll update the PR.

We need to update the web site in a separate issue: http://yetus.apache.org/mailinglists/

Comment (aw-was-here) [reply .asf.yaml]: We sent gitbox stuff there but the JIRA and other ASF stuff goes to those MLs to make things a bit less noisy. We were one of the first projects to go to the gitbox stuff before this config file existed.

**Review by aw-was-here — APPROVED** [review]

**Comment by aajisaka** [comment]
Thank you @aw-was-here

**Pull request merged**